HGustavs/LenaSYS
# Sectioned.php now works with old database

This issue uses try/catch statements to work around the database queries, then uses isset() to determine if a column is present or not in the answer received. I'd also like to add that the only relevant changes are to sectionedservice.php, the other changes are due to a merge with my previous issue branch due to the changes to navheader.php being needed to load the page.

**Contact me or the current leaders for access to the old database.**

**Testing instructions:**

- Initially, check out any courses page on the current database to ensure it still works.
- Then run the database file to get the old database (if using phpmyadmin, click your database and select import)
- Following this, go into "sessions.php" and go to the function "isSuperUser" (line 426) and add "return true;" to the top of the function. This is done due to the account we use no longer existing, hence the admin buttons would not show otherwise.
- Then simply go to the following link:                                                                                                                                           http://localhost/LenaSYS/DuggaSys/sectioned.php?courseid=1&coursename=Webbutveckling%20-%20Programmering%20av%20mobila%20applikationer&coursevers=928742                                                                                    or any other correctly put together link and test away.
